src/js/components/PackageDetailVersionsDropdown.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+
+import { FormGroup, FieldSelect } from "#SRC/js/components/form";
+
+const METHODS_TO_BIND = ["handleVersionChange"];
+
+class PackageDetailVersionsDropdown extends Component {
+  constructor() {
+    super();
+
+    METHODS_TO_BIND.forEach(method => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  handleVersionChange(event) {
+    const packageVersion = event.target.value;
+    const originalPath = global.location.hash.split("?")[0];
+    const selectedVersion = `${originalPath}?version=${packageVersion}`;
+
+    global.location.replace(selectedVersion);
+  }
+
+  render() {
+    const { cosmosPackage, cosmosPackageVersions } = this.props;
+    const currentVersion = cosmosPackage.getCurrentVersion();
+    const packageVersions = cosmosPackageVersions.getAllVersions();
+    const sortedPackageVersions = Object.keys(packageVersions).sort();
```

Comment:
It looks like the intention here is that we want to sort such that the latest version is last. Then, we should short using the values (instead of keys) from the response object. For example, chronos is 

`{2.5.0: "5", 3.0.1: "4", 3.0.0: "2", 2.4.0: "0"}` (case here is 2.5.0 is actually latest)

Also, if you look at the sort order for confluent-kafka package, the sort order is not correct since `1.1.4-3.0.0` should come before `1.1.19.1-3.2.2`.

Reply:
Correct, I'm going to change to sort by the values. Thanks.

Reply:
fixed. even though the `values` can also be not in "order"